components/Navigation/Navigation.tsx

```diff
@@ -110,19 +110,14 @@ export const NavigationContainer = styled('div', {
 export type NavigationContainerVariants = VariantProps<typeof NavigationContainer>;
 export type NavigationContainerProps = NavigationContainerVariants & {};
 
-const elevation = Object.keys(elevationVariant).map((elevationKey) => ({
-  ...elevationVariant[Number(elevationKey)],
-  bc: `$${Number(elevationKey) < 10 ? '0' : ''}${elevationKey}dp`,
-})) as any;
-
```

Comment:
@paulocfjunior :information_source: I permitted myself to remove the extra computation for `bc` in `Navigation` L.115